cmd/pmm-configurator/main.go

```diff
@@ -11,15 +11,18 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/percona/pmm-manage/configurator/config"
+	"github.com/percona/pmm-manage/configurator/sshkey"
 	"github.com/percona/pmm-manage/configurator/user"
 )
 
 var c config.PMMConfig
+var SSHKey sshkey.Handler
```

Comment:
I don't want to rework whole application architecture right now, can it be postponed?